ipu-plugin/pkg/ipuplugin/deviceplugin.go

```diff
@@ -0,0 +1,71 @@
+package ipuplugin
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
+
+	pb2 "github.com/openshift/dpu-operator/tree/main/dpu-api/gen"
+	"google.golang.org/protobuf/types/known/emptypb"
```

Comment:
Dpu operator defines its own empty message and we should use that one

Reply:
fixed it